documentation/docs/steps/cloudFoundryDeploy.md

```diff
@@ -65,6 +66,7 @@ Deployment can be done
 
 * `deployTool` defines the tool which should be used for deployment.
 * `deployType` defines the type of deployment, either `standard` deployment which results in a system downtime or a zero-downtime `blue-green` deployment.
+* `keepOldInstance` defines if in case of a `blue-green` deployment the old instance will be deleted or stopped
```

Comment:
I would wish for a more explicit description of the default behaviour, and what flipping that switch means.